app/assets/stylesheets/application.css.scss

```diff
@@ -1,5 +1,6 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
+@import "bootstrap-extend";
```

Comment:
Could we put our override styles under `shared/` folder, so that we know this is our override, otherwise it looks like a bootstrap built-in.

```
@import "shared/bootstrap-extension`
```

or

```
@import "shared/bootstrap-overrides`
```